docs/reference/setup/install/deb.asciidoc

```diff
@@ -2,9 +2,16 @@
 === Install Elasticsearch with Debian Package
 
 The Debian package for Elasticsearch can be <<install-deb,downloaded from our website>>
-or from our  <<deb-repo,APT repository>>. It can be used to install
+or from our <<deb-repo,APT repository>>. It can be used to install
 Elasticsearch on any Debian-based system such as Debian and Ubuntu.
 
+This package is free to use under the Elastic license. It contains open source 
```

Comment:
I wonder if there is value in having this whole section as a fragment, and including via `include::license_stuffs`? it might cut down on the number of places to update going forward, if there are links and such we need to change? If done as an include, we could use that include at least 5 times in the files in this PR seemingly?

Everything else LGTM 💯